tasks/config.yml

```diff
@@ -22,3 +22,11 @@
   with_items:
     - 'alertmanager.yml'
     - 'amtool.yml'
+
+- name: Create template file
+  copy:
+    src: 'custom.tmpl'
+    dest: '{{ alertmanager_service_path }}/conf/templates/custom.tmpl'
```

Comment:
Alertmanager keeps most of the stuff in one file:
https://github.com/prometheus/alertmanager/blob/main/template/default.tmpl

So I thought one file is good enough. It can have all kind of templates.